pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-deutsche-post.php

```diff
@@ -424,36 +440,36 @@ public function order_items_table( $items, $current_wc_order = null ) {
 		<table class="widefat striped" id="pr_dhl_order_items_table">
 			<thead>
 			<tr>
-				<th><?php _e( 'Item', 'dhl-for-woocommerce' ) ?></th>
-				<th><?php _e( 'Actions', 'dhl-for-woocommerce' ) ?></th>
+				<th><?php esc_html_e( 'Item', 'dhl-for-woocommerce' ); ?></th>
+				<th><?php esc_html_e( 'Actions', 'dhl-for-woocommerce' ); ?></th>
 			</tr>
 			</thead>
 			<tbody>
-			<?php echo implode( '', $table_rows ) ?>
+			<?php echo implode( '', wp_kses_post( $table_rows ) ); ?>
```

Comment:
This change caused Fatal error
<img width="538" alt="image" src="https://github.com/user-attachments/assets/1130340e-c58d-4665-9993-84c2e0980d45">

$table_rows is array of HTML table rows , and no need to use **wp_kses_post** here

@OlfatProgressus